### PR TITLE
Feature/apbs 1.4.1 intel 2015a

### PR DESCRIPTION
**Pull request by molden**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+sources = [SOURCELOWER_TGZ]
+
+# parallel build fails
+parallel = 1
+
+builddependencies = [
+    ('CMake', '3.2.2'),
+]
+
+preconfigopts = 'cd apbs && '
```
Comment (boegel) [inline easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb]: use `start_dir = 'apbs'` instead?

File: easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+homepage = 'http://www.poissonboltzmann.org/apbs'
+description = """ APBS is a software package for modeling biomolecular solvation 
+ through solution of the Poisson-Boltzmann equation (PBE), one of the most popular 
+ continuum models for describing electrostatic interactions between molecular solutes 
+ in salty, aqueous media. """
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+#git clone -b 1.4.1-binary-release  https://github.com/Electrostatics/apbs-pdb2pqr
+#create your own tarball
```
Comment (boegel) [inline easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb]: use the official location for sources, see http://sourceforge.net/projects/apbs/files/apbs/apbs-1.4.1/ (via http://www.poissonboltzmann.org/docs/downloads/#downloads) ?

File: easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+parallel = 1
+
+builddependencies = [
+    ('CMake', '3.2.2'),
+]
+
+preconfigopts = 'cd apbs && '
+
+configopts = '-DENABLE_BEM=off   '
+configopts += '-DENABLE_iAPBS=on '
```
Comment (boegel) [inline easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb]: collapse these to a single line?

File: easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+prebuildopts = 'cd apbs && '
+
+preinstallopts = 'cd apbs && '
```
Comment (boegel) [inline easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb]: these are no longer needed if you're using `start_dir`

File: easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb (hunk 2 above)
Comment (molden) [inline easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb]: Hmm, I don't see the sources only binaries ...

Comment (boegel) [inline easybuild/easyconfigs/a/APBS/APBS-1.4.1-intel-2015a.eb]: Argh, what a bunch of idiots... >_<

Can we figured out which git commit ID matches 1.4.1 exactly? Maybe https://github.com/Electrostatics/apbs-pdb2pqr/commit/74fcb8676de69ed04ddab8976a8b05a6caaf4d65 ? Or maybe one of the older commits in https://github.com/Electrostatics/apbs-pdb2pqr/commits/1.4.1-binary-release ?

Then you can do something like:

```python
source_urls = ['https://github.com/Electrostatics/apbs-pdb2pqr/archive/']
sources = ['74fcb867.tar.gz']
```

**Comment by boegel** [comment]
@molden ping on remarks?

**Comment by migueldiascosta** [comment]
@molden, this PR is being closed for the following reason(s): no activity for > 1 year.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**